content/03.methods.md

```diff
@@ -87,6 +87,50 @@ $$\hat{\theta} = argmin_{\theta} -\sum_{\Lambda} log \sum_{Y}P(\Lambda, Y)$$
 
 In the framework we used predictions from the generative model, $\hat{Y} = P(Y \mid \Lambda)$, as training classes for our dataset [@doi:10.14778/3157794.3157797; @doi:10.1145/3209889.3209898]. 
 
+### Discriminative Model
+
+The discriminative model is a neural network, which we trained using labels predicted from the generative model.
+The expectation is that the discriminative model can improve upon the generative model's predictions by incorporating textual features along with generated training labels.
+We used a piecewise convolutional neural network that contains multiple kernel filters as our discriminative model.
+This network uses multiple filters with fixed widths of 300 dimensions and a fixed height of 7 (Figure {@fig:convolutional_network}).
```

Comment:
```suggestion
We built a network with multiple filters using a fixed width of 300 dimensions and a fixed height of 7 (Figure {@fig:convolutional_network}).
```
Is dimensions the right term here - is this the window size for words?

Reply:
This is the window size for the words. Will change dimensions to something a bit more straight forward.